dirty_equals/_datetime.py

```diff
@@ -178,3 +178,99 @@ def __init__(
         )
         if tz is not None:
             self._repr_kwargs['tz'] = tz
+
+
+class IsDate(IsNumeric[date]):
+    """
+    Check if the value is a date, and matches the given conditions.
+    """
+
+    allowed_types = date
+
+    def __init__(
+        self,
+        *,
+        approx: Optional[date] = None,
+        delta: Optional[Union[timedelta, int]] = None,
+        gt: Optional[date] = None,
+        lt: Optional[date] = None,
+        ge: Optional[date] = None,
+        le: Optional[date] = None,
+        unix_number: bool = False,
+        iso_string: bool = False,
+        format_string: Optional[str] = None,
+    ):
+
+        """
```

Comment:
you need to add `IsDate` to the docs, probably add it after [this line](https://github.com/samuelcolvin/dirty-equals/blob/fe7122e306cabdcd27d2545bd88b79dfba6820fb/docs/types/datetime.md?plain=1#L48).

Reply:
only docs remained in todo. I'll do it in an available time.

Reply:
this still needs adding, should be two lines.